buildSrc/src/main/groovy/org/opensearch/hadoop/gradle/fixture/OpenSearchFixturePlugin.groovy

```diff
@@ -60,8 +60,8 @@ class OpenSearchFixturePlugin implements Plugin<Project> {
     }
 
     private static void createClusterFor(TaskCollection<StandaloneRestIntegTestTask> integrationTests, Project project, String version) {
-        def clustersContainer = project.extensions.getByName(TestClustersPlugin.EXTENSION_NAME) as NamedDomainObjectContainer<ElasticsearchCluster>
-        def integTestCluster = clustersContainer.create("integTest") { ElasticsearchCluster cluster ->
+        def clustersContainer = project.extensions.getByName(TestClustersPlugin.EXTENSION_NAME) as NamedDomainObjectContainer<OpenSearchCluster>
+        def integTestCluster = clustersContainer.create("integTest") { OpenSearchCluster cluster ->
             cluster.version = version
             cluster.testDistribution = TestDistribution.DEFAULT
```

Comment:
Ah, this is the problem right here, OpenSearch changed from `DEFAULT` to `INTEG_TEST` [here](https://github.com/opensearch-project/OpenSearch/blob/3fede8be3cf133a3835d4ad38479628ad04240be/buildSrc/src/main/java/org/opensearch/gradle/testclusters/TestDistribution.java#L38). I've raised a PR on your fork @nknize